lib/utils.js

```diff
@@ -20,7 +20,14 @@ const labelParser = function (labels) {
 }
 /* Fingerprinting */
 const shortHash = require('short-hash')
-const fingerPrint = function (text, hex) {
+const xxh = require('xxhashjs')
+const fingerPrint = function (text, hex, alg) {
+  alg = alg || process.env.HASH || 'short-hash'
+  switch (alg) {
+    case 'XXH':
```

Comment:
naming is not consistent:

* `short-hash`
* `XXH`

For example, `XXH` could be renamed to `xxhash64`